packages/api/src/pubsub.ts

```diff
@@ -89,7 +103,17 @@ export const createPubSubClient = (): PubsubClient => {
       })
 
       if (await findFeatureByName(FeatureName.AISummaries, userId)) {
-        await enqueueAISummarizeJob({
+        // await enqueueAISummarizeJob({
+        //   userId,
+        //   libraryItemId,
+        // })
+      }
+
+      if (
```

Comment:
@sywhb this is the other issue we kind of have with better job handling. Might need a new way of adding jobs based on library item.